### PR TITLE
Adding the macos notification update post

### PR DESCRIPTION
**Pull request by gauntface**
cc @jpmedley @Meggin @kosamari @petele 

Would be good to get this landed as soon as possible.

### CODE REVIEW

File: src/content/en/updates/2017/04/native-mac-os-notifications.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+directly by the MacOS native notification system instead of Chrome's own system.
+
+This change makes Chrome on MacOS feel much better integrated into the platform
+and fixes a number of long standing bugs, such as Chrome not respecting the
+system Do Not Disturb setting.
+
+Below we'll look at the differences this change introduces to the existing
+API's.
+
+## Notification Center
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/native-mac-os-notifications.md]: Please make headings ## and ### sentence case.

File: src/content/en/updates/2017/04/native-mac-os-notifications.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+One of the benefits (and feature requests from users) of this change is that
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/native-mac-os-notifications.md]: (and feature requests from users) doesn't seem to fit logically or grammatically in the sentence. Can you take another shot at this.

File: src/content/en/updates/2017/04/native-mac-os-notifications.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: As of Chrome version 59, notifications will be displayed native notifications on Mac OS X.
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/native-mac-os-notifications.md]: Remove 'version'. 'Chrome 59' is sufficient.

Also, I think 'displayed native notifications' should be 'displayed as native notifications'.

Comment (jpmedley) [inline src/content/en/updates/2017/04/native-mac-os-notifications.md]: And be sure to copy these changes to line 9.

File: src/content/en/updates/2017/04/native-mac-os-notifications.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+  <figcaption>
+    Before and after of notification icons displayed by Chrome vs displayed by
+    Mac OS X.
+  </figcaption>
+</figure>
+
+### Action Icons
+
+Before this change action buttons and icons would be displayed in the
+notification. With the native notifications the action button icons will not be
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/native-mac-os-notifications.md]: 'With the native' => 'With native'.

File: src/content/en/updates/2017/04/native-mac-os-notifications.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+used and the user will need to hover over the notification and select the "More"
+button to see the available actions.
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image02.png">
+  <figcaption>
+    Before and after of notification action buttons with icons displayed by
+    Chrome vs displayed by Mac OS X.
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/native-mac-os-notifications.md]: Per devsite style guide, figure captions do not end with a period. Please remove throughout.

File: src/content/en/updates/2017/04/native-mac-os-notifications.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+### Images
+
+The image option will no longer be supported on OS X. If you define an image
+property the notification will still be displayed, but it will ignore the image
+parameter (See example below).
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image03.png">
+  <figcaption>
+    Before and after of notification image displayed by Chrome vs displayed by Mac OS X.
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/native-mac-os-notifications.md]: You could just simplify to 'Before and after for Chrome on Mac.'  Everything I've removed to get that is provided by the context. Regardless, the old sentence is bit of a word salad and should be revised.

File: src/content/en/updates/2017/04/native-mac-os-notifications.md [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+templates](https://developer.chrome.com/apps/notifications#type-TemplateType)
+which will behave differently with this change.
+
+The image notification template will no longer show the image. You should ensure
+that images are supplemental and not required to be useful to your users.
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image04.png">
+  <figcaption>
+    Before and after of example of the image template in the chrome.notification API.
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/native-mac-os-notifications.md]: Maybe 'Before and after for image templates in the chrome.notification API. It's the phrase 'Before and after of example' that's striking me as awkward. Please address that throughout.

File: src/content/en/updates/2017/04/native-mac-os-notifications.md [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+to indicate the progress instead of a progress bar.
+
+<figure>
+  <img src="/web/updates/images/2017/04/macos-notifications/image06.png">
+  <figcaption>
+    Before and after of example of the progress template in the chrome.notification API.
+  </figcaption>
+</figure>
+
+The other difference in API / notification UI is the appIconMarkUrl which will
```
Comment (jpmedley) [inline src/content/en/updates/2017/04/native-mac-os-notifications.md]: Backticks on `appIconMarkUrl`. This also needs to be fixed in the caption below.

**Comment by gauntface** [comment]
@jpmedley please review

**Comment by petele** [comment]
will push live shortly.

**Pull request merged**